spec/hydra/works/models/concerns/generic_file/characterization_spec.rb

```diff
@@ -0,0 +1,286 @@
+require 'spec_helper'
+require 'support/generic_file_helper'
+
+describe Hydra::Works::GenericFile::Characterization do
+  let(:demo_class) do
+    Class.new(Hydra::Works::GenericFile::Base) do
+      include Hydra::Works::GenericFile::Characterization
```

Comment:
In the context of let's block, `described_class` is undefined.  
```
Failure/Error: include described_class
     NameError:
       undefined local variable or method `described_class' for #<Class:0x007feea58766a8>
     # ./spec/hydra/works/models/concerns/generic_file/characterization_spec.rb:8:in `block (3 levels) in <top (required)>'
```
It's the reason there is a rubocop exception for RSpec/DescribedClass that excludes this spec.